lint/eslint.stylish-formatter.js

```diff
@@ -0,0 +1,141 @@
+// Fork of 'stylish' plugin that prints relative paths.
+// This allows an editor to navigate to the location of the lint warning even though we present
+// eslint with paths underneath a bazel sandbox folder.
+// from https://github.com/eslint/eslint/blob/331cf62024b6c7ad4067c14c593f116576c3c861/lib/cli-engine/formatters/stylish.js
+/**
+ * @fileoverview Stylish reporter
+ * @author Sindre Sorhus
+ */
+"use strict";
+
+/**
+ * LOCAL MODIFICATION:
+ * The three eslint dependencies should be loaded from the user's node_modules tree, not from rules_lint.
+ */
+
+// This script is used as a command-line flag to eslint, so the command line is "node eslint.js --format this_script.js"
+// That means we can grab the path of the eslint entry point, which is beneath its node modules tree.
+const eslintEntry = process.argv[1];
+// Walk up the tree to the location where eslint normally roots the searchPath of its require() calls
+const searchPath = eslintEntry.substring(
+  0,
+  eslintEntry.length - "/node_modules/eslint/bin/eslint.js".length
```

Comment:
It might be that some versions of eslint don't have the entry point as `bin/eslint.js` and this would fail. Most likely it is consistent but not necessarily going to stay that way. Safest way to proceed is probably to find the last `/node_modules/` in the path and strip to that point. If there is no `/node_modules/` segment then just error out.

Reply:
Yeah I was thinking that for a moment, wow I'm lazy this week 🌵